.github/workflows/mindsdb.yml

```diff
@@ -68,6 +68,11 @@ jobs:
         CHECK_FOR_UPDATES: False
         DB_MACHINE_KEY: ${{secrets.DB_MACHINE_KEY}}
         DATABASE_CREDENTIALS: ${{secrets.DATABASE_CREDENTIALS}}
+    - name: Install latest version from pypi (just testing)
+      run: |
+        sleep 20
+        pip install mindsdb
+        pip show mindsdb
```

Comment:
Was this for testing when you made the PR, if so please remove before merging. all else looks fine.

Reply:
Done